### DISCUSSION TITLE
blog/linkding-self-hosted-bookmark-manager/

### DISCUSSION THREAD

**Discussion by giscus** [AkashRajpurohit/AkashRajpurohit#49] [Comments]
# blog/linkding-self-hosted-bookmark-manager/

In this post I will show you how to setup Linkding, a self hosted bookmark manager that I am personally using for a while now and I am very happy with it. I will also share with you my setup and how I am using it.

https://akashrajpurohit.com/blog/linkding-self-hosted-bookmark-manager/

**Comment by andrewg78** [comment]
Hi, thx for this article :) Some comments from my end:
1.For Android there is a native app for linkding - https://github.com/fibelatti/pinboard-kotlin
2. It is also worth to mention, that there is an integration with SingleFile extension to save a complete page when creating a bookmark.
3, LInk to linkding_shortcut.json is dead (404) but since there is a Pinkt App, HTTP Shortcuts is not needed anymore.

**Reply by AkashRajpurohit** [reply]
Nice, thanks for mentioning these pointers, will update the blog to reflect about these :)
